server/graphql/core/src/loader/loader_registry.rs

```diff
@@ -35,224 +35,224 @@ pub async fn get_loaders(
         ItemLoader {
             connection_manager: connection_manager.clone(),
         },
-        async_std::task::spawn,
+        tokio::spawn,
```

Comment:
I am doing some test on a docker image to see if this makes a difference, may switch to actix_rt::spawn since this will run in actix_web context, will comment further

Reply:
makes not difference with actix_rt::spawn or tokio::spawn, for some reason I trust tokio::spawn more. Data loaders had issues for me with 'delay' using future-rs, if you interested: https://github.com/async-graphql/async-graphql/issues/1716